pkg/server/grpc/tracee.go

```diff
@@ -3,18 +3,50 @@ package grpc
 import (
 	"context"
 	"fmt"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/mennanov/fmutils"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 
 	tracee "github.com/aquasecurity/tracee/pkg/ebpf"
 	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/streams"
 	"github.com/aquasecurity/tracee/pkg/version"
 	pb "github.com/aquasecurity/tracee/types/api/v1beta1"
+	"github.com/aquasecurity/tracee/types/trace"
 )
 
 type TraceeService struct {
 	pb.UnimplementedTraceeServiceServer
 	tracee *tracee.Tracee
 }
 
+func (s *TraceeService) StreamEvents(in *pb.StreamEventsRequest, grpcStream pb.TraceeService_StreamEventsServer) error {
+	stream, err := s.getStream(in.Policies)
```

Comment:
nit: I think it will be better to inline getStream since (1) it is short, and (2) it then makes it clear why we have Unsubscribe below